Sources/AppcuesKit/Data/Analytics/AutoPropertyDecorator.swift

```diff
@@ -39,10 +39,23 @@ internal class AutoPropertyDecorator: AnalyticsDecorating {
     }
 
     func decorate(_ tracking: TrackingUpdate) -> TrackingUpdate {
+        // Early return for group updates
```

Comment:
I found it confusing that there was returns mixed in below, so moving those to the top.